lib/ansible/plugins/lookup/hashi_vault.py

```diff
@@ -8,7 +8,8 @@
 DOCUMENTATION = """
   lookup: hashi_vault
   author: Jonathan Davila <jdavila(at)ansible.com>
-  version_added: "2.0"
+  contributors: Drew Mullen <mullen.drew@gmail.com>
+  version_added: "2.1"
```

Comment:
Why had this changed?

Reply:
the version? because we're adding new features. is that not right?

contributors? because i dont know the format well enough. ill remove it

Reply:
Use this:
```yaml
version_added: "2.0"
author:
  - Jonathan Davila <jdavila(at)ansible.com>
  - Drew Mullen (@drewmullen) <mullen.drew@gmail.com>
```

`version_added` is the version of Ansible the module was added.